### PR TITLE
83 integrate twitter auth

### PR DESCRIPTION
**Pull request by andrii-zaharovskyi**
Cose #83

### CODE REVIEW

File: EventsExpress/ClientApp/src/config.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 ﻿export default {
   "FACEBOOK_CLIENT_ID": "2741444662848625",
-  "GOOGLE_CLIENT_ID": "3429495463-t3uma25030ea0q819p2f600bi9j1869l.apps.googleusercontent.com"
+  "GOOGLE_CLIENT_ID": "3429495463-t3uma25030ea0q819p2f600bi9j1869l.apps.googleusercontent.com",
+  "TWITTER_CALLBACK_URL": "https://localhost:5002/Authentication/TwitterLogin",
```
Comment (iuriikhrystiuk) [inline EventsExpress/ClientApp/src/config.js]: This will not work properly on stage and prod, since it will not be redirected to localhost there. Consider using relative URL here.

File: EventsExpress/ClientApp/src/services/EventsExpressService.js [hunk 2]
```diff
@@ -130,6 +131,13 @@ export default class EventsExpressService {
             : await res.json();
     }
 
+    setTwitterLoging = async (data) => {
```
Comment (iuriikhrystiuk) [inline EventsExpress/ClientApp/src/services/EventsExpressService.js]: Please use `setTwitterLogin` not  `setTwitterLogin`*g*

**Review by iuriikhrystiuk — CHANGES_REQUESTED** [review]

File: EventsExpress/ClientApp/src/config.js [hunk 3]
```diff
@@ -1,4 +1,6 @@
 ﻿export default {
-  "FACEBOOK_CLIENT_ID": "2741444662848625",
-  "GOOGLE_CLIENT_ID": "3429495463-t3uma25030ea0q819p2f600bi9j1869l.apps.googleusercontent.com"
+  "GOOGLE_CLIENT_ID": "3429495463-t3uma25030ea0q819p2f600bi9j1869l.apps.googleusercontent.com",
+  "TWITTER_CALLBACK_URL": "https://eventsexpress-test.azurewebsites.net/Authentication/TwitterLogin",
```
Comment (iuriikhrystiuk) [inline EventsExpress/ClientApp/src/config.js]: Do these changes work locally? Can you not use the relative url here?

File: EventsExpress/EventsExpress.csproj [hunk 4]
```diff
@@ -82,7 +82,7 @@
     <!-- Include the newly-built files in the publish output -->
     <ItemGroup>
       <DistFiles Include="$(SpaRoot)build\**; $(SpaRoot)build-ssr\**" />
-      <ResolvedFileToPublish Include="@(DistFiles->'%(FullPath)')" Exclude="@(ResolvedFileToPublish)">
+      <ResolvedFileToPublish Include="@(DistFiles-&gt;'%(FullPath)')" Exclude="@(ResolvedFileToPublish)">
```
Comment (iuriikhrystiuk) [inline EventsExpress/EventsExpress.csproj]: Why is this change required?

Comment (andrii-zaharovskyi) [reply EventsExpress/EventsExpress.csproj]: My IDE makes this change automatically. This has been fixed.

File: EventsExpress/ClientApp/src/config.js (hunk 3 above)
Comment (andrii-zaharovskyi) [reply EventsExpress/ClientApp/src/config.js]: Reworked using the relative URL.

File: EventsExpress/ClientApp/src/services/EventsExpressService.js (hunk 2 above)
Comment (andrii-zaharovskyi) [reply EventsExpress/ClientApp/src/services/EventsExpressService.js]: Typo "g" fixed.

File: EventsExpress/ClientApp/.vscode/launch.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (iuriikhrystiuk) [inline EventsExpress/ClientApp/.vscode/launch.json]: Please modify `.gitignore` to ignore .vscode folders and do not iclude such in sources.

File: EventsExpress/ClientApp/package-lock.json [hunk 6]
```diff
@@ -5520,6 +5520,11 @@
         "randomfill": "^1.0.3"
```
Comment (iuriikhrystiuk) [inline EventsExpress/ClientApp/package-lock.json]: Please remove this file from PR.

File: EventsExpress/EventsExpress.xml [hunk 7]
```diff
@@ -15,21 +15,29 @@
         </member>
```
Comment (iuriikhrystiuk) [inline EventsExpress/EventsExpress.xml]: Please remove this file from PR.

File: EventsExpress/ClientApp/.vscode/launch.json (hunk 5 above)
Comment (andrii-zaharovskyi) [reply EventsExpress/ClientApp/.vscode/launch.json]: Accidentally added `VSCode` files have been removed and the `.gitignore` file has been changed.

File: EventsExpress/ClientApp/package-lock.json (hunk 6 above)
Comment (andrii-zaharovskyi) [reply EventsExpress/ClientApp/package-lock.json]: The file has been removed.

File: EventsExpress/EventsExpress.xml (hunk 7 above)
Comment (andrii-zaharovskyi) [reply EventsExpress/EventsExpress.xml]: The file has been removed.

**Review by iuriikhrystiuk — APPROVED** [review]
